src/runtimes/rego/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod context_aware;
+mod errors;
```

Comment:
don't we have to make that a `pub mod errors` to allow consumers of the library to use our errors?

Reply:
Scratch my comment, these are internal errors so far.